### PR TITLE
website: whats-new v9 updates

### PR DESCRIPTION
**Pull request by felixpalmer**
Open to discussion on the content, just wanted to get the ball rolling

Preview deployed at: https://felixpalmer.github.io/deck.gl/docs/whats-new

### CODE REVIEW

File: docs/whats-new.md [hunk 1]
```diff
@@ -2,30 +2,55 @@
...
+
+<table style={{border: 0}} align="center">
+  <tbody>
+    <tr>
+      <td>
+        <img style={{maxHeight:200}} src="https://github.com/visgl/deck.gl-data/blob/master/images/whats-new/collision-filter-extension.gif?raw=true" />
+        <p><i>WebGPU ready</i></p>
+      </td>
+      <td>
+        <img style={{maxHeight:200}} src="https://github.com/visgl/deck.gl-data/blob/master/images/whats-new/terrain-extension.gif?raw=true" />
```
Comment (felixpalmer) [inline docs/whats-new.md:19]: I'm aware that the highlighted features are things that I've worked on & had high visibility of. Happy to replace with other things, I just think it is nice to have this visual section at the top

File: docs/whats-new.md [hunk 2]
```diff
@@ -2,30 +2,55 @@
...
 
+### Category filtering in DataFilterExtension
+
+The [DataFilterExtension](./api-reference/extensions/data-filter-extension) has been enhanced to support filtering based on categories. A new accessor, [getFilterCategory](./api-reference/extensions/data-filter-extension), is added which allows filtering using the new [filterCategories](./api-reference/extensions/data-filter-extension#filtercategories).
+
+### FirstPersonController
+
+The behavior of the [FirstPersonController](./api-reference/core/first-person-controller) has been modified to more accurately reflect the motion of a human, making it more intuitive to navigate a 3D space. In addition, panning is supported.
+
 ### Breaking Changes
```
Comment (Pessimistress) [inline docs/whats-new.md:42]: This should be in Upgrade Guide instead

**Review by Pessimistress — COMMENTED** [review]
The highlights IMO:

- GPU readiness (no visuals, but we should expand the session to elaborate 1) why it's the future 2) that v9 is supporting WebGL2 and WebGPU at the same time 3) WebGPU is coming in 9.1
- Packages are now fully ESM compatible
- TypeScript is official (GIF showing type hint during coding)
- Widgets module (screenshot)
- CategoryFilter (GIF)

File: docs/whats-new.md (hunk 2 above)
Comment (chrisgervang) [reply docs/whats-new.md:42]: This also feels like feature highlight that could have a GIF showing a couple new keyboard controls (+ to go vertical, arrows to walk around). I think OSX screen capture records key strokes.

**Comment by felixpalmer** [comment]
Typescript code preview:

deck-typescript-v9

File: docs/whats-new.md [hunk 3]
```diff
@@ -2,30 +2,75 @@
...
+
+### Category filtering in DataFilterExtension
+
+The [DataFilterExtension](./api-reference/extensions/data-filter-extension) has been enhanced to support filtering based on categories. A new accessor, [getFilterCategory](./api-reference/extensions/data-filter-extension), is added which allows filtering using the new [filterCategories](./api-reference/extensions/data-filter-extension#filtercategories).
+
+### FirstPersonController
+
+The behavior of the [FirstPersonController](./api-reference/core/first-person-controller) has been modified to more accurately reflect the motion of a human, making it more intuitive to navigate a 3D space. In addition, panning is supported.
+
+### `@deck.gl/carto`
```
Comment (alasarr) [inline docs/whats-new.md:62]: @felixpalmer I have add this. Let me know what do you think

**Review by alasarr — APPROVED** [review]

**Comment by chrisgervang** [comment]
FPV-trimmed

File: docs/whats-new.md [hunk 4]
```diff
@@ -2,30 +2,97 @@
...
+      </td>
+      <td>
+        <img style={{maxHeight:200}} src="https://github.com/visgl/deck.gl/assets/453755/e05863e2-777f-46fa-a04a-1bf4aa0625e2" />
+        <p><i>Category filtering</i></p>
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+### New module: @deck.gl/widgets
```
Comment (chrisgervang) [inline docs/whats-new.md:28]: I've added widgets section and a GIF to FPV. Happy to make adjustments.

**Review by chrisgervang — APPROVED** [review]

File: docs/whats-new.md (hunk 3 above)
Comment (felixpalmer) [reply docs/whats-new.md:84]: Looks good, I think it makes sense to link to our docs rather than expand here

**Review by Pessimistress — APPROVED** [review]

File: docs/whats-new.md [hunk 5]
```diff
@@ -2,30 +2,97 @@
...
+</table>
+
+### `@deck.gl/carto`
+
+- New CARTO module architecture: Sources and Layers replace the CartoLayer. Read more in the [Upgrade Guide](./upgrade-guide.md#deckglcarto).
 
-Changes should mostly impact custom layers that use luma.gl and GLSL shaders directly. More information about breaking changes and migration strategies will be provided as part of deck.gl v9 documentation.
+- Public preview for Boundary Sources (experimental): easily aggregate large data volumes aggregated into common or custom geographic boundaries, such as postal codes. Learn more about Boundaries in the CARTO [documentation](https://docs.carto.com/carto-for-developers/guides/use-boundaries-in-your-application).
 
-- `@deck.gl/mapbox` - `MapboxLayer` has been remove. Use `MapboxOverlay` instead.
```
Comment (chrisgervang) [inline docs/whats-new.md:19]: Where did this get moved too? Previously it was under breaking changes and should remain

Comment (felixpalmer) [reply docs/whats-new.md:19]: https://github.com/visgl/deck.gl/blob/fe0afd3a8a0411d6b3a9387ad9bdc5f7bd5f2518/docs/upgrade-guide.md#deckglmapbox

**Pull request merged**